scripts/ci/docker-compose/local.yml

```diff
@@ -53,7 +53,9 @@ services:
       - ../../../setup.cfg:/opt/airflow/setup.cfg:cached
       - ../../../setup.py:/opt/airflow/setup.py:cached
       - ../../../tests:/opt/airflow/tests:cached
+      - ../../../tests:/opt/airflow-backport/tests:cached
       - ../../../tmp:/opt/airflow/tmp:cached
+      - ${HOME}/.config:/root/.config
```

Comment:
Why do you want this directory to be available in the container? It contains very sensitive data, e.g. oAuth.2.0 for gcloud and hub.

Reply:
Precisely for that reason. This way for example gcloud authentication is mapped to the container, so you can use your host gcloud authentication to run gcloud commands in docker

This is only for CI/development/local execution so there is no risk in passing it to inside the docker. Both host and docker are controlled by the developer.

Reply:
Personally, I don't trust any code that runs in the container. In my opinion, there should be no sensitive data in the container, because users who are unaware that some sensitive data is available in the container may hurt themselves. Could you add it as an optional feature?

There is a technical possibility to do this in a way that does not allow the user to hurt himself. We can talk about it tomorrow.

Reply:
I am not sure I am following this line of thinking.  Not sure what "hurt themselves" mean in this case  - but happy to discuss it tomorrow.